README.md

```diff
@@ -478,8 +507,8 @@ Offline Installations   | ✓     | x
   For example, `npm` allows `file` dependencies with the `cachito_npm_file_deps_allowlist`
   configuration. `gomod` allows this through the `go.mod` replace directive.
 * **Offline Installations** - The dependencies can be installed solely with the contents of the
-  bundle. This is true for the `gomod` package manager, however, the `npm` package manager relies
-  on Nexus to be online and properly configured by Cachito.
+  bundle. This is true for the `gomod` package manager, however, the `npm` and `pip` package
+  managers rely on Nexus to be online and properly configured by Cachito.
 
 ### gomod
```

Comment:
do you ever provide information on the offline installation feature for the gomod package manager? I assume that this is achieved with the `gomod-vendor` flag (i.e. replacing the vendor directory)... but I have not seen it yet. Since this is unique to golang, I would be interested to know a little background behind _why_ it is.

Reply:
resolved in the other thread